docs/manual/common/guide/devmode/KafkaServer.md

```diff
@@ -47,7 +47,7 @@ In sbt:
 
 ## JVM options
 
-The Kafka server is run on a separate process, and a JVM is started with sensible memory defaults. However, if the default JVM options don't suit you, you can override them by adding the following in your build.
+The Kafka server is running on a separate process, and a JVM is started with sensible memory defaults. However, if the default JVM options don't suit you, you can override them by adding the following in your build.
```

Comment:
```suggestion
The Kafka server runs on a separate process, and a JVM is started with sensible memory defaults. However, if the default JVM options don't suit you, you can override them by adding the following in your build.
```

Reply:
(I think I'm the original author of the sentence)

IIRC my intention for this sentence was:

```suggestion
The Kafka server is ran on a separate process, and a JVM is started with sensible memory defaults. However, if the default JVM options don't suit you, you can override them by adding the following in your build.
```

as in `the batman logo is projected in the clouds`.

But I'm not native speaker of the english language and as a reader I find @marcospereira's suggestion clearer.

Reply:
"is ran" isn't right.  And "is running" is less good, and just wrong with "a JVM is started".

The original "is run" works fine with "a JVM is started", but so does "runs".

If anything the "on" should be "in": "The Kafka server runs in a separate process".